cli/ioctl/cmd/account/accountbalance.go

```diff
@@ -7,73 +7,47 @@
 package account
 
 import (
-	"context"
 	"fmt"
+	"strings"
 
 	"github.com/spf13/cobra"
 	"go.uber.org/zap"
-	grpc "google.golang.org/grpc"
 
-	"github.com/iotexproject/iotex-core/cli/ioctl/cmd/config"
 	"github.com/iotexproject/iotex-core/pkg/log"
-	pb "github.com/iotexproject/iotex-core/protogen/iotexapi"
 )
 
 // TODO: use wallet config later
 var configAddress = "ioaddress"
 
 // accountBalanceCmd represents the account balance command
 var accountBalanceCmd = &cobra.Command{
-	Use:   "balance",
+	Use:   "balance []address",
 	Short: "Get balance of an account",
 	Run: func(cmd *cobra.Command, args []string) {
 		fmt.Println(balance(args))
 	},
 }
 
-func init() {
-	AccountCmd.AddCommand(accountBalanceCmd)
-}
-
-// balance gets balance of an IoTex Blockchian address
+// Balance gets balance of an IoTex blockchain address
 func balance(args []string) string {
-	endpoint := config.GetEndpoint()
-	if endpoint == config.ErrEmptyEndpoint {
-		log.L().Error(config.ErrEmptyEndpoint)
-		return "use \"ioctl config set endpoint\" to config endpoint first."
-	}
-	conn, err := grpc.Dial(endpoint, grpc.WithInsecure())
-	if err != nil {
-		log.L().Error("failed to connect to server", zap.Error(err))
-		return err.Error()
-	}
-	defer conn.Close()
-	cli := pb.NewAPIServiceClient(conn)
-	ctx := context.Background()
-
-	request := pb.GetAccountRequest{}
-	res := ""
+	lines := make([]string, 0)
 	if len(args) == 0 {
-		request.Address = configAddress
-		response, err := cli.GetAccount(ctx, &request)
+		accountMeta, err := GetAccountMeta(configAddress)
 		if err != nil {
-			log.L().Error("cannot get account from "+request.Address, zap.Error(err))
+			log.L().Error("cannot get account from "+configAddress, zap.Error(err))
```

Comment:
No need to log error here if you return error message to the caller. Otherwise, the error will be printed twice, then?